python/pip_install/repositories.bzl

```diff
@@ -16,8 +16,8 @@ _RULE_DEPS = [
     ),
     (
         "pypi__setuptools",
-        "https://files.pythonhosted.org/packages/ab/b5/3679d7c98be5b65fa5522671ef437b792d909cf3908ba54fe9eca5d2a766/setuptools-44.1.0-py2.py3-none-any.whl",
-        "992728077ca19db6598072414fb83e0a284aca1253aaf2e24bb1e55ee6db1a30",
+        "https://files.pythonhosted.org/packages/70/06/849cc805ac6332210083f2114a95b22ee252ce81ed4e1be4f1d2b87c9108/setuptools-54.0.0-py3-none-any.whl",
```

Comment:
@aaliddell 

> Version 44 was the last to support both Python 2 & 3. So until rules_python fully drops Python 2 support, it may be stuck with this version.

does that also preclude this change ?

Reply:
Perhaps, but I think the new `pip_install` is only intended to support Python 3 (#409). The issue seems to be that there's also the older packaging rules still present, which support Python 2.

Reply:
it would be really helpful if this were included from our pov, but my bazel knowledge is a bit limited so im not totally sure of the effect